mesheryctl/cmd/perf.go

```diff
@@ -79,17 +84,26 @@ var perfCmd = &cobra.Command{
 		if len(testURL) > 0 {
 			postData = postData + "\nendpoint_url: " + testURL
 		} else {
-			println("Error: Please enter a TestURL")
+			println("\nError: Please enter a test URL")
```

Comment:
Do we require and additional new line, given that Println already prints in a new line?

Reply:
An extra blank line was seperating the two error message which were quite clear to read. 
Also, if spacing is issue we can amend it anytime.